### ISSUE TITLE
Add option to get a notification after all apps are available

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#8708]
It would be very handy if you can get a notification, when all apps have their update, so you don't need to check the settings page all the time.

Copied from https://github.com/nextcloud/caniupdate/issues/3

**Comment by szaimen** [comment]
I think update notifications for apps are already implemented -> closing.

**Comment by nickvergessen** [comment]
yeah well the problem is the update might be incompatible with your current version, so it's not suggested and no notification will be generated.
